apps/n8n/1.70.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   homeassistant:
-    image: n8nio/n8n:1.70.0
+    image: n8nio/n8n:1.70.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is no clear issue or difference highlighted. However, the changes mentioned here (replacing '1.70' with '1.70.1') might suggest an improvement to accommodate any updates that were made since version 1.70. If you need more specific advice from my perspective, please share what modifications you are considering.